assignment_2nd/src/main/java/com/pironeer/week2_1/controller/CommentController.java

```diff
@@ -0,0 +1,54 @@
+package com.pironeer.week2_1.controller;
+
+import com.pironeer.week2_1.dto.request.CommentCreateRequest;
+import com.pironeer.week2_1.dto.request.CommentUpdateRequest;
+import com.pironeer.week2_1.dto.response.CommentResponse;
+import com.pironeer.week2_1.service.CommentService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/comment")
+@Tag(name = "댓글(Comment)")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @PostMapping
+    @Operation(summary = "댓글 작성")
+    public ResponseEntity<?> create(@RequestBody CommentCreateRequest request){
```

Comment:
```suggestion
    public ResponseEntity<?> create(@Valid @RequestBody CommentCreateRequest request){
```

유효성 검사를 위해 애노테이션을 추가해 주세요!